config-linux.md

```diff
@@ -466,19 +466,37 @@ The following parameters can be specified to set up the controller:
 The following parameters can be specified for the container:
 
 * **`l3CacheSchema`** *(string, OPTIONAL)* - specifies the schema for L3 cache id and capacity bitmask (CBM).
-    If `l3CacheSchema` is set, runtimes MUST write the value to the `schemata` file in the `<container-id>` directory discussed in `intelRdt`.
+* **`memBwSchema`** *(array of objects, OPTIONAL)* - specifies the schema of memory bandwidth (b/w) percentage per L3 cache id.
```

Comment:
why not make this simpler with just or something of the like:

```
"bandwidth" : {
    "id": 0,
    "value" 70,
}
```

you don't have to prefix everything when they are part of an object, its implied that it is an attribute of the containing object.

Reply:
@crosbymichael 
> why not make this simpler with just or something of the like:

It makes sense. Thank you.